bundles/org.openhab.binding.heos/src/main/java/org/openhab/binding/heos/internal/resources/Telnet.java

```diff
@@ -231,22 +198,16 @@ private int concatReadLineResult(String value) {
      * @throws IOException
      */
     public void disconnect() throws IOException {
```

Comment:
Corrected Telnet disconnect logic, the stream should not be closed by the caller.